### ISSUE TITLE
Dependency updates - analyzer

### ISSUE THREAD

**Issue by talamaska** [kwado-tech/l10n_mapper#17]
I get a warning when running the generator
about analyzer version being less then analyzer ^6.4.1
when I try to manually set it in my pubspec I get 
```
Resolving dependencies...
Because every version of l10n_mapper_generator depends on analyzer ^5.2.0 and giftmate depends on analyzer ^6.4.1, l10n_mapper_generator is forbidden.
So, because giftmate depends on l10n_mapper_generator any, version solving failed.
```
So please take a look at this
```
Your current `analyzer` version may not fully support your current SDK version.
Analyzer language version: 3.1.0
SDK language version: 3.3.0
```

**Comment by kwado-tech** [comment]
Hey @talamaska I see your dart sdk constraints is >=3.3.0. What is your current flutter version? 

Will update l10n-mapper to match the the latest dart version.

**Comment by talamaska** [comment]
Sorry, forgot to add flutter doctor
```
[√] Flutter (Channel stable, 3.19.5, on Microsoft Windows [Version 10.0.22631.3296], locale en-US)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.9.3)
[√] Android Studio (version 2023.2)
[√] VS Code (version 1.88.0)
[√] Connected device (3 available)
[√] Network resources
```

I guess flutter 3.19 comes with dart 3.3

**Comment by kwado-tech** [comment]
This is resolved in version 2.0.1
